regulations/views/preamble.py

```diff
@@ -272,8 +272,12 @@ def get_preamble(doc_number):
         preamble['children'].insert(0, intro['tree'])
     intro['meta'] = convert_to_python(intro.get('meta', {}))
     if 'comments_close' in intro['meta']:
-        intro['meta']['days_remaining'] = 1 + (
-            intro['meta']['comments_close'].date() - date.today()).days
+        today = date.today()
+        close_date = intro['meta']['comments_close'].date()
+        intro['meta']['days_remaining'] = 1 + (close_date - today).days
+        intro['meta']['accepts_comments'] = close_date >= today
+    else:
+        intro['meta']['accepts_comments'] = True
```

Comment:
Let's default to `False`. Then our viewer would work on non-proposed rules, too.